README.md

```diff
@@ -1,183 +1,181 @@
 its queue will reach capacity and workers adding new inputs will block
-until the worker catches up.  This back pressure prevents processes and
+until the worker catches up. This back pressure prevents processes and
 their mailboxes from growing unboundedly when workers outrun their
 followers.
 
-** Coordinators
+## Coordinators
 
-For each fitting to be executed, there is a *coordinator* for the
-cluster as a whole, separate from the per-vnode worker. Each
-coordinator is responsible for conveying its fitting's behavior as
-well as handling the coordination necessary at the end of input.
+For each fitting to be executed, there is a **coordinator** for the
+cluster as a whole, separate from the per-vnode worker. Each coordinator
+is responsible for conveying its fitting's behavior as well as handling
+the coordination necessary at the end of input.
 
-** Sinks
+## Sinks
 
-Each pipeline has a *sink*, where all of the final output is sent
+Each pipeline has a **sink**, where all of the final output is sent
 (and, optionally, log messages as well). By default, the sink and the
 client process that initiated the pipeline are one and the same.
 
-* Step by Step
+# Step by Step
 
-** Start the pipeline
+## Start the pipeline
 
-The first step in performing an operation on =riak_pipe= is to set up
+The first step in performing an operation on `riak_pipe` is to set up
 the coordinators to manage the pipeline. Each coordinator knows the
 details for the behavior of its fitting, the partition choice for
-distributing output to the workers for the next fitting, and the =pid=
+distributing output to the workers for the next fitting, and the `pid`
 of the next coordinator.
 
-=riak_pipe= exposes a client interface (see the [[API]] section) for
-starting these coordinators.  It begins by starting a "builder"
-process, which then starts and monitors each of the coordinators,
-starting with the final one (so that its =pid= can be passed to its
-predecessor).  When all of the coordinators have started, a structure
-capturing their details is returned to the client.
-
-** Send inputs
-
-All inputs are sent directly to the vnodes that will process them.
-They are tagged with the =pid= of the coordinator that holds the
-details of how they will be processed.  A vnode will add each input to
-a queue it maintains for that fitting.  The input-sending operation
-will block until the input has been added to the queue.
-
-Sending inputs directly to vnodes is done instead of sending every
-input to the coordinator and distributing from there, in order to
-remove a bottleneck and reduce messaging.
-
-Blocking input-sending until the enqueueing process completes is used
-as a means of providing back pressure for workers that would otherwise
-overwhelm their downstream partners.  When a queue is already at its
+`riak_pipe` exposes a client interface (see the <span
+class="spurious-link" target="API">*API*</span> section) for starting
```

Comment:
```suggestion
`riak_pipe` exposes a client interface (see the [API](#api) section) for starting
```

This was meant to be a link to the API section in this document. I think the syntax I've suggested does this, but I haven't checked it.

Reply:
Do we need to retain the italic formatting? if so, I think it should look like this:
```suggestion
`riak_pipe` exposes a client interface (see the [*API*](#api) section) for starting
```

Reply:
I think it was the Pandoc that added the italic formatting, probably to help people notice the "spurious-link". I like it, though.